README.md

```diff
@@ -1,8 +1,36 @@
 ## Orange DAM PHP API client
 
-The [Orange DAM](https://www.orangelogic.com/products/digital-asset-management-system) APIs allow developers to safely and securely extend the capabilities of their Orange DAM platform. Through calls to its API endpoints, an Orange DAM site can be manipulated and integrated with other software.
+The [Orange DAM](https://www.orangelogic.com/products/digital-asset-management-system) APIs allow developers to extend the capabilities of their Orange digital asset management platform. Through calls to its API endpoints, an Orange DAM site can be manipulated and integrated with a CMS or other software/workflows.
 
-__NOTICE: This library is in pre-release and under active development. It should be considered a preview and not ready for production use.__
+This library powers the **[Orange DAM Drupal module](https://www.drupal.org/project/orange_dam)**. For integration with other CMS platforms, see [Future Development](#future-development).
+
+This library enables the following:
+
+- Authentication
+- Search content.
+- Media retrieval/import.
+- Access Keyword data/activity.
+- Retrieve media captions.
+- List deleted objects.
+
+### Future Development
+
+The Orange DAM API is extensive and this library just scratches the surface of available functionality. Some _ideas_ about where this library and/or the tools that consume it could go next include:
+
+- WordPress plugin.
+- Enable upload of content to Orange DAM.
+- Access file metadata.
+- Document support.
+- Create links between documents.
+- Integration with Figma asset browser.
+- Integration with Adobe Lightroom.
+- Integration with Adobe InDesign.
+- PDF conversion.
+- Extract waveforms from audio content.
+
+_NOTE:_ This library and the associated module(s) that depend on it provide the foundation for a custom integration with Orange DAM. Custom development work will be needed to create a data model in your application, map your data, and/or integrate with the events this module offers.
```

Comment:
> Anyone looking at a PHP library should already understand that this doesn't do anything on it's own or at least not have that assumption since it isn't tied to a CMS. I realize I'm making my own assumption there, but leaving this just feels like it's answering a question that no one is asking to me.

I totally get where you are coming from, but I think this could be valuable for someone who ends up on this repository (maybe via search) and _doesn't_ understand all of the things you noted above, and I don't think it takes anything away for the folks who do already grok those points.